n26/library/templates/authoring/picklist_member_remove.html

```
@@ -0,0 +1,36 @@
+{% comment %}
+The question asked before an option is taken off a list, at an address
+of its own. A page rather than a prompt: it survives a reload, Back is a
+real answer, and it works with scripting switched off. Nothing here
+changes anything — the form's POST is the act.
+
+The first line says the one fact the row's Remove control cannot: what
+goes is this list's place for the option, never the option itself.
+{% endcomment %}
+{% block head_title %}Remove {{ label }}{% endblock head_title %}
+{% block content %}
+    <c-n26.form-page action="{% url 'authoring-picklist-member-remove' thing.pk %}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)